### PR TITLE
Comply with Keep a Changelog guideline

### PR DESCRIPTION
**Pull request by yudai-nkt**
- Add an ad hoc guideline so that contributors can easily understand where to document their changes
- Fix incorrect size of heading and inconsistent markup for unordered list

### CODE REVIEW

**Comment by Aerijo** [comment]
I don't see anything wrong with the changes you've made.

Before merging though, would it be possible to add the date of release to each version? This seems to be a recommended practice; see [1](https://www.gnu.org/prep/standards/html_node/Style-of-Change-Logs.html#Style-of-Change-Logs) & [2](https://github.com/olivierlacan/keep-a-changelog/blob/master/CHANGELOG.md) . I would make a separate pull, but it might be quicker and cleaner just to add it to this one.

**Comment by yudai-nkt** [comment]
I used a quite similar format in my `language-tex` (I don't remember where I learned the format, though), but didn't know there was such kind of guideline. This is really cool and I will follow Keep a Changelog in this PR.

**Comment by yudai-nkt** [comment]
I fully re-wrote `CHANGELOG.md` and the edit might be a somewhat radical change. Can I ask for a review, @Aerijo?

Beside the review, I have one question to discuss. When we document release dates, which time zone should be assumed? UTC+09:00 (Japan Standard Time) is currently assumed but I believe this is not convenient for you. Reasonable alternatives are

- UTC: it's easier to derive what time it is in UTC than in other time zones
- UTC-08:00 (Pacific Standard Time): GitHub, Inc. is there

I prefer UTC to PST. Which one do you prefer?

**Comment by yudai-nkt** [comment]
Damn, I said "colons" in the commit message but they are _periods_ lol. We need to fix it when this PR get squashed into the `master` branch.

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,105 +1,145 @@
...
 
-## 1.1.0
+### Changed
+- Set 'softWrap' to be true by default in LaTeX files.
 
-#### Added
-- Grammar for gnuplot environments
+## [1.1.1] - 2017-08-19
+### Fixed
+- Highliting for `lstlisting` environment without explicit language specification ([#121](https://github.com/area/language-latex/pull/121)).
```
Comment (Aerijo) [inline CHANGELOG.md]: Typo - Highlighting

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,105 +1,145 @@
...
-- Grammar for deprecated `eqnarray` environment
-- Grammar for Beamer's `\usetheme` and its components
-- Grammar for single quotation
+[#126](https://github.com/area/language-latex/pull/126)).
+- Support for highlighting long lines ([#131](https://github.com/area/language-latex/pull/131)).
+
+### Fixed
+- Fix plural forms for cite commands ([#14](https://github.com/area/language-latex/issues/14)).
+- Add correct highlighting for textcite and parencite commands ([#53](https://github.com/area/language-latex/pull/53)).
+- Highliting for `lstlisting` environment without explicit language specification ([#121](https://github.com/area/language-latex/pull/121)).
```
Comment (Aerijo) [inline CHANGELOG.md]: Typo - Highlighting. Seems like the only other one though.

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,105 +1,145 @@
...
+
+### Removed
+- Highlighting for quoation punctuations ([#96](https://github.com/area/language-latex/pull/96)).
+
+## [1.0.0] - 2016-10-01
+### Added
+- workaround for listing envs without language specified.
```
Comment (Aerijo) [inline CHANGELOG.md]: Capitalise the 'w'

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,105 +1,145 @@
...
 - Fixing math snippet (thanks to [Zach Tidwell](https://github.com/hzach)).
-- \begin{lstlisting} syntax highlighting fix (thanks to Martin Molnar).
+- Grammar for `lstlisting` environment ([#41](https://github.com/area/language-latex/issues/41) and [#43](https://github.com/area/language-latex/pull/43)).
+
+## [0.6.1] - 2015-04-14
+### Fixed
+- Missing backslashed in snippets ([#30](https://github.com/area/language-latex/issues/30) and [#32](https://github.com/area/language-latex/pull/32)).
+
+## [0.6.0] - 2015-04-11
+__Disclaimer__: Version 0.5.0 is missing due to ower's mistake. See [`3d233d1`](https://github.com/area/language-latex/commit/3d233d1adf72d693cece41ccb55b83e983d8308c) for some details.
```
Comment (Aerijo) [inline CHANGELOG.md]: "owner's"

File: CHANGELOG.md [hunk 5]
```diff
@@ -1,105 +1,145 @@
...
+
+### Added
+- Grammar for `luacode` and `luacode*` environment ([#20](https://github.com/area/language-latex/pull/20)).
+
+### Changed
+- Rename the depricated `scoped-properties` directory to `settings`.
```
Comment (Aerijo) [inline CHANGELOG.md]: Deprecated

File: CHANGELOG.md [hunk 6]
```diff
@@ -1,105 +1,145 @@
...
+- Unwanted line breaks in `tabular` environment ([#4](https://github.com/area/language-latex/issues/4) and [#5](https://github.com/area/language-latex/pull/5)).
 
-## 0.2.0
-* Snippets improvement.
+## 0.1.0 - 2014-02-27
+- Initial release.
 
-## 0.1.0
-* Initial release.
+[Unreleased]: https://github.com/area/language-latex/compare/v0.1.0...HEAD
```
Comment (Aerijo) [inline CHANGELOG.md]: Is this right? I don't see much value in comparing the unreleased changes with the original release.

**Comment by Aerijo** [comment]
And for timezone, I think UTC is best too. PST is nothing special for me, so a standard timezone fits for everyone better.

File: CHANGELOG.md (hunk 6 above)
Comment (yudai-nkt) [reply CHANGELOG.md]: This was a typo also; it should be a comparison between v1.1.1 and HEAD.

**Comment by yudai-nkt** [comment]
Thanks for your proofreading, I think all typos are fixed in 4ff2ee8. I also changed some of the release dates that are in different dates between UTC and my timezone.

**Pull request merged**